app/main.py

```diff
@@ -1,8 +1,59 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+
+        Person.people[name] = self
+
+
+    def set_spouse(self, spouse):
+        if self.age < spouse.age:
+            self.wife = spouse
+            spouse.husband = self
+        else:
+            self.husband = spouse
+            spouse.wife = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_instances = []
+    for person in people:
+        new_person = Person(person["name"], person["age"])
```

Comment:
The `new_person` instance is created but not added to the `person_instances` list. Consider appending `new_person` to `person_instances` here.